docs/content/guide/bootstrap.ngdoc

```diff
@@ -142,6 +142,17 @@ This is the sequence that your code should follow:
   2. Call {@link angular.bootstrap} to {@link compiler compile} the element into an
   executable, bi-directionally bound application.
 
+## Things to keep in mind
+
+There a few things to keep in mind regardless of automatic or manual bootstrapping:
+
+- There is no explicit support for bootstrapping more than one AngularJS applications on a page, and
+  doing this may cause bugs and/or memory leaks.
+- Likewise, apps cannot be nested within each other.
```

Comment:
Yeah, you can only auto-bootstrap one app. You need to manaully bootstrap the rest.
I haven't used it with any complex nested app, but simple have worked without problems.

But still, it's untested :disappointed: